docs/getting_started/setup_your_environment.md

```diff
@@ -144,6 +144,30 @@ project (`npm init -y` as necessary), then add the following block to your
 }
 ```
 
+#### LSP clients
+
+Deno has builtin support for the
+[Language server protocol](https://langserver.org).
+
+If your editor supports the LSP, you can add Deno as a language server for
+TypeScript and JavaScript.
+
+The editor can start the server with `deno lsp`.
```

Comment:
```suggestion
The language server can be started with `deno lsp`.
```

Reply:
Eh, I feel like people that do not know how the LSP works will interpret this as "Type `deno lsp` into the terminal".

Reply:
Sure